validator/src/consensus/signing/client.test.ts

```diff
@@ -124,18 +130,18 @@ describe("signing", () => {
 	it("e2e signing flow", async () => {
 		const nonceCommitmentsEvents: {
 			groupId: GroupId;
-			index: bigint;
+			signerId: bigint;
```

Comment:
Just so I am sure, `signerId` is a `FROST.Identifier` correct?

Reply:
yes, and it should actually be of type ParticipantId here (because that is my type for FROST.Identifier)